src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -14,6 +14,9 @@ use super::{
     Machine, PlaceTy, OpTy, InterpretCx,
 };
 
+mod type_name;
+
+pub use type_name::*;
```

Comment:
Seems a bit funny to have that here when the intrinsic is not even available in CTFE?^^